### PR TITLE
ref(echart-tooltip): Add indent hack

### PR DESCRIPTION
**Pull request by priscilawebdev**
used in https://github.com/getsentry/getsentry/pull/6397

### CODE REVIEW

File: static/app/components/charts/components/tooltip.tsx [hunk 1]
```diff
@@ -181,12 +181,28 @@ function getFormatter({
...
+
+          if (formattedLabel.includes('indent:')) {
+            label = formattedLabel.split('indent:')[1];
+            indent = true;
+          }
+
           const value = valueFormatter(getSeriesValue(s, 1), s.seriesName);
-          return `<div><span class="tooltip-label">${s.marker} <strong>${formattedLabel}</strong></span> ${value}</div>`;
+
+          console.log('value', value);
```
Comment (k-fish) [inline static/app/components/charts/components/tooltip.tsx:195]: console.log?

**Review by k-fish — COMMENTED** [review]
Rather then rely on modifying the seriesName and using a string hack, I think instead you should just build an object for additional tooltip options for each series name, then pass that to your chart in the `tooltip` param -> that gets passed as props to Tooltip -> you can use that inside the formatter.

**Review by markstory — APPROVED** [review]
Looks good to me. Using the series names to indent is a simple interface, and preferrable to using series indexes imo.

**Review by k-fish — APPROVED** [review]
Yeah, this looks better 👍

**Pull request merged**